public/js/data.js

```diff
@@ -354,95 +384,12 @@ function deleteCurrentCircuit() {
         showMessage("Circuit was not deleted")
 }
 
-// Generates JSON of the entire project
-function generateSaveData(name) {
```

Comment:
Why are there so many changes here? Is it just movement of code or have you made actual changes?